mlflow/cli.py

```diff
@@ -53,7 +53,8 @@ def cli():
                    "corresponding entry point as command-line arguments in the form `--name value`")
 @click.option("--docker-args", "-A", metavar="NAME=VALUE", multiple=True,
               help="A `docker run` flag or argument, of the form -A name=value. Where `name` "
-              "will then be propagated as `docker run --name value`.")
+              "will then be propagated as `docker run --name value`. "
+              "Pass -A tty=true to allocate a pseudo-TTY.")
```

Comment:
@Zethson Instead of renaming / aliasing the `-t` argument of `docker run` as `-tty`, can we continue to use `-t`? This way, we don't have to do the special casing in `mlflow.projects.__init__`. This will likely be clearer to users who are familiar with the `-t` flag in Docker.

Reply:
Sure. Will not look that descriptive: `-A t=true`, but fine with me if you prefer it.

Reply:
Awesome! Thanks!